src/callout.tsx

```diff
@@ -51,7 +52,10 @@ const Callout: React.FC<Props> = ({
     role,
 }) => {
     const variant = useThemeVariant();
-    const {texts} = useTheme();
+    const {
+        texts,
+        i18n: {locale},
+    } = useTheme();
```

Comment:
As you can see we now require to get the `locale` and import a `translate` function to get the translation. 
Also, the `locale` must be converted to language inside the `translate` function.

This can be improved.

`useTheme` could provide the `translate` function and the language can be precalculated in the context creation, so no extra imports (apart of the tokens) would be needed. 

Unless you think a better approach, I will make this change

Reply:
choose one?

```js
const {t} = useTheme(); // I prefer this one. `t` is commonly used for translations
const {translate} = useTheme(); 
const {i18n: {t}} = useTheme();
const {i18n: {translate}} = useTheme();
```

Reply:
I like `const {translate} = useTheme();` but `t` is ok too
You could have a different hook just for translations too `const translate = useTranslate()`

Reply:
I prefer to keep it in the same useTheme hook. This way we avoid an additional import and the useTheme is required anyway to get the text overrides